app/javascript/widget/views/Home.vue

```diff
@@ -163,6 +170,10 @@ export default {
         this.showAttachmentError = false;
       }, 3000);
     });
+    bus.$on(BUS_EVENTS.START_NEW_CONVERSATION, () => {
+      this.isOnCollapsedView = true;
```

Comment:
Do we need to set this?

Reply:
Never mind, we do.

Reply:
Yes.  It is used for skipping the team availability page on a new conversation. https://github.com/chatwoot/chatwoot/blob/186c3e225b4e50035df2a7e7494f1e3c8efac138/app/javascript/widget/views/Home.vue#L133